aladdin/lib/arg_tools.py

```diff
@@ -55,3 +56,37 @@ def add_command(parser, cmd=bash_cmd, help_msg=bash_help["help"]):
             sub_parser.set_defaults(func=lambda args: bash_wrapper())
         commands.append((bash_cmd, add_command))
     return commands
+
+
+HELM_OPTION_PARSER = argparse.ArgumentParser(add_help=False)
+HELM_OPTION_PARSER.add_argument(
+    "--chart", help="which chart in the project to use, defaults to the project name"
+)
+HELM_OPTION_PARSER.add_argument(
+    "--dry-run",
+    "-d",
+    action="store_true",
+    help="Run the helm as test and don't actually run it",
+)
+HELM_OPTION_PARSER.add_argument(
+    "--force-helm",
+    action="store_true",
+    help="Have helm force resource update through delete/recreate if needed",
+)
+HELM_OPTION_PARSER.add_argument(
+    "--set-override-values",
+    default=[],
+    nargs="+",
+    help=(
+        "override values in the values file. "
+        "Syntax: --set-override-values key1=value1 key2=value2 customValues.yaml ..."
+    ),
+)
+
+COMMON_OPTION_PARSER = argparse.ArgumentParser(add_help=False)
+COMMON_OPTION_PARSER.add_argument(
+    "--namespace",
+    "-n",
+    default=os.getenv("NAMESPACE", "default"),
+    help="namespace name, defaults current: [{}]".format(os.getenv("NAMESPACE", "default")),
+)
```

Comment:
abstracted common arguments into these base parsers. I opted out of updating all the places where we could use `COMMON_OPTION_PARSER` for now (since it could be used in _all_ the python commands).